airflow/jobs/base_job.py

```diff
@@ -73,6 +72,17 @@ class BaseJob(Base, LoggingMixin):
         Index('idx_job_state_heartbeat', state, latest_heartbeat),
     )
 
+    task_instances_enqueued = relationship(
+        TaskInstance,
+        primaryjoin=id == TaskInstance.queued_by_job_id,
+        foreign_keys=id,
+        backref=backref('queued_by_job', uselist=False),
+    )
+    """TaskInstances which have been enqueued by this Job.
+
```

Comment:
```suggestion
```

I think ?

Reply:
Nope, the empty line is there to separate "headline" from detail for this doc.